src/components/PeopleLoader/PeopleLoader.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useCallback, useEffect } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+const findParent = (
+  people: Person[],
+  parentName: string,
+): Person | undefined => {
+  return people.find(parent => parent.name === parentName);
+};
+
+export const PeopleLoader = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const loadPeople = useCallback(async () => {
+    setIsLoading(true);
+    try {
+      const PeopleFromServer = await getPeople();
+
+      setPeople(PeopleFromServer);
+    } catch {
+      setError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  }, []);
```

Comment:
https://react.dev/reference/react/useCallback

Read more about useCallback and when to use it. There's a specific case when you're using the function returned from useCallback inside the useEffect that's mentioned in the React documentation on useCallback.